caravel/views.py

```diff
@@ -860,15 +864,40 @@ def pre_update(self, obj):
     def pre_delete(self, obj):
         check_ownership(obj)
 
+    @action("mulexport", "Export", "Export dashboards?", "fa-database")
+    def mulexport(self, items):
+        ids = ''.join('&id={}'.format(d.id) for d in items)
+        return redirect(
+            '/dashboardmodelview/export_dashboards_form?{}'.format(ids[1:]))
+
+    @expose("/export_dashboards_form")
+    def download_dashboards(self):
+        if request.args.get('action') == 'go':
+            ids = request.args.getlist('id')
+            return Response(
+                models.Dashboard.export_dashboards(ids),
+                headers=generate_download_headers("pickle"),
+                mimetype="application/text")
+        return self.render_template(
+            'caravel/export_dashboards.html',
+            dashboards_url='/dashboardmodelview/list'
+        )
+
 
 appbuilder.add_view(
     DashboardModelView,
     "Dashboards",
-    label=__("Dashboards"),
+    label=__("Dashboards List"),
     icon="fa-dashboard",
-    category="",
+    category='Dashboards',
     category_icon='',)
 
+appbuilder.add_link(
+    'Import Dashboards',
+    href='/caravel/import_dashboards',
+    icon="fa-flask",
```

Comment:
maybe this one:
http://fontawesome.io/icon/files-o/

![screenshot 2016-09-28 14 42 37](https://cloud.githubusercontent.com/assets/130878/18933494/efbe2602-8589-11e6-8d68-c7549dc6b5ce.png)